src/interface/interface.tsx

```diff
@@ -0,0 +1,25 @@
+//todo 인터페이스 정의
+export interface ITodo {
+  id: number;
+  text: string;
+  isCompleted: boolean;
+}
+
+//handleTodoInput 인터페이스 정의
+export interface ITodoInputTypes {
```

Comment:
`props`의 인터페이스는 보통 이름에 props가 들어갑니다
```suggestion
export interface ITodoInputProps {
```

Reply:
그렇군요!! 인터페이스를 처음 만들어봐서 네이밍할 때 고민했었는데 감사합니다!! 반영해서 수정해보겠습니다!